### ISSUE TITLE
Not showing up?

### ISSUE THREAD

**Issue by Razunter** [C1rdec/Poe-Lurker#107]
**Describe the bug**
App does not have any visible UI even with "always visible" on. PoE in borderless fullscreen.
Sound and tray icon do work.

 - Version v1.5.10 and earlier (tried before too)

>  - Provide Log (C:\Users\[user]\AppData\Local\PoeLurker\app-[version]\logs)

There are no "logs" folder.

**Additional context**
High DPI windows font setting, 3440x1440 resolution, two monitors. Windows 10 x64 1909.
Other similar apps work fine.

**Comment by C1rdec** [comment]
If you turn on the debug mode can you see Green Square on the screen?

**Comment by Razunter** [comment]
Nope: https://i.imgur.com/uBNLvXx.jpg

**Comment by C1rdec** [comment]
Can we setup a discord call

**Comment by C1rdec** [comment]
https://discord.gg/B8ZwZV

**Comment by Razunter** [comment]
TLDR: don't override app's DPI scaling setting.
